pkg/cedar/intake/client.go

```diff
@@ -162,3 +164,99 @@ func (c *Client) publishIntakeObject(ctx context.Context, model translation.Inta
 
 	return nil
 }
+
+func (c *Client) PublishOnSchedule(ctx context.Context, store *storage.Store, dayOfWeek time.Weekday, hourInUTC int) {
+	logger := appcontext.ZLogger(ctx)
+	if hourInUTC > 24 || hourInUTC < 0 {
+		logger.Error("incorrect hour given for publish schedule, use int between 0 and 24")
+		return
+	}
+	if !c.enabled {
+		logger.Info("CEDAR intake publish schedule is disabled")
+		return
+	}
```

Comment:
I think most of our testing will just have to come from validating in lower level environments.

We _might_ want to consider some other control (a different feature flag? Env var?) that defines whether or not this behavior is enabled in a given environment, since we might want to push this out to `prod` without immediately enabling it (and being forced to disable _all_ CEDAR publishing)

Reply:
That seems like a pretty good idea to me!

Reply:
@mynar7 @ClayBenson94 , I added on an envrc for `CEDAR_INTAKE_PUBLISHER_ENABLED` and confirmed that this is being checked locally. Can you both check this out and see if there is any other place where things might have been missed? I think that we can deploy to LLEs with it enabled, and turn on in PROD after CEDAR validates the changes.